src/Discord/functions.php

```diff
@@ -110,3 +110,41 @@ function getColor($color = 0): int
 
 	return 0;
 }
+
+/**
+ * Replacement for Illuminate\Support\Str::contains.
+ *
+ * @param string $string The string to check.
+ * @param array $matches Array containing one or more phrases to match.
+ *
+ * @return bool
+ */
+function contains(string $string, array $matches): bool
+{
+    foreeach($matches as $match) {
```

Comment:
`foreeach`

Reply:
lol yeah, forgot that, had it fixed in my running tree, not just yet here...